src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -0,0 +1,75 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+public class TestPtvConsts {
+  
+  public final static String[] ORGANIZATIONS = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2386)